lib/SPIRV/SPIRVBuiltinHelper.cpp

```diff
@@ -183,9 +183,21 @@ BuiltinCallMutator &BuiltinCallMutator::replaceArg(unsigned Index,
 BuiltinCallMutator &BuiltinCallMutator::removeArg(unsigned Index) {
   // If the argument being dropped is the last one, there is nothing to move, so
   // just remove the attributes.
-  if (Index == Args.size() - 1)
-    Attrs = Attrs.removeParamAttributes(CI->getContext(), Index);
-  else
+  if (Index == Args.size() - 1) {
+    // TODO: Remove this workaround when LLVM fixes potential bug on
```

Comment:
Perhaps it would be useful to include a link to the issue.
```suggestion
    // TODO: Remove this workaround when LLVM fixes https://github.com/llvm/llvm-project/issues/59746 on
```